content/css/decred.css

```diff
@@ -643,9 +643,6 @@ body {
   margin-left: 20px;
   float: right;
   clear: left;
-  background-image: url('../images/example-bg.png');
```

Comment:
Might as well remove example-bg.png unless you anticipate needing it for development in the future.